### PR TITLE
chore: rename feature rbac to template_rbac

### PR DESCRIPTION
**Pull request by deansheather**

### CODE REVIEW

File: enterprise/coderd/license/license.go [hunk 1]
```diff
@@ -176,7 +176,7 @@ type Features struct {
 	BrowserOnly    int64 `json:"browser_only"`
 	SCIM           int64 `json:"scim"`
 	WorkspaceQuota int64 `json:"workspace_quota"`
-	RBAC           int64 `json:"rbac"`
+	TemplateRBAC   int64 `json:"template_rbac"`
```
Comment (kylecarbs) [inline enterprise/coderd/license/license.go:179]: This will need to be changed in `coder/license` too

**Review by BrunoQuaresma — APPROVED** [review]
@deansheather I made a minor fix on the FE. Code LGTM!

File: enterprise/coderd/license/license.go (hunk 1 above)
Comment (deansheather) [reply enterprise/coderd/license/license.go:179]: Which files do you mean exactly?

Comment (kylecarbs) [reply enterprise/coderd/license/license.go:179]: Well, right now it's authorized as `rbac` in that repo, so a license couldn't be generated that would make this feature enabled.

Comment (bpmct) [reply enterprise/coderd/license/license.go:179]: https://github.com/coder/license/pull/9

Comment (deansheather) [reply enterprise/coderd/license/license.go:179]: wait this is for both template ACLs and groups? shouldn't we split it out?

Comment (sreya) [reply enterprise/coderd/license/license.go:179]: groups are pretty useless with template acl and vice versa

**Pull request merged**